src/components/PlaylistForm.vue

```diff
@@ -0,0 +1,220 @@
+            </VBtn>
+          </VCol>
+        </VRow>
+        <VRow no-gutters align="center" justify="center">
+          <VCol md="9" sm="10" cols="12" @change.once="isDirty = true">
+            <h4 class="text-h6 mt-6 ml-4">
+              {{ $tc("music.playlist.items", 2) }}
+            </h4>
+            <VSimpleTable>
+              <thead>
+                <tr>
+                  <th style="width: 1px" class="text-center">Sort</th>
+                  <th>
+                    {{
+                      newPlaylist.playlist_type === "artist"
+                        ? $t("music.name")
+                        : $t("music.title")
+                    }}
+                  </th>
+                  <th></th>
+                </tr>
+              </thead>
+              <Draggable
+                tag="tbody"
+                v-model="newPlaylist.item_ids"
+                handle="[data-draggable=handle]"
+              >
+                <tr
+                  v-for="(item_id, index) of newPlaylist.item_ids"
+                  :key="item_id"
+                >
+                  <td class="text-no-wrap">
+                    <VBtn small icon text class="" data-draggable="handle">
+                      <VIcon>mdi-drag-horizontal-variant</VIcon>
+                    </VBtn>
+                    {{ index }}
+                  </td>
+                  <td class="play-queue__cell">
+                    {{
+                      newPlaylist.playlist_type === "artist"
+                        ? items[item_id].name
+                        : items[item_id].title
+                    }}
+                  </td>
+                  <td class="text-right">
+                    <VBtn
+                      small
+                      icon
+                      text
+                      class="ma-2 red--text"
+                      @click="() => newPlaylist.item_ids.splice(index, 1)"
+                    >
+                      <VIcon>mdi-close</VIcon>
+                    </VBtn>
+                  </td>
+                </tr>
+              </Draggable>
+            </VSimpleTable>
+          </VCol>
+        </VRow>
+      </VForm>
+    </VCol>
+  </VRow>
+</template>
+
+<script>
+import { mapActions, mapState } from "vuex";
+import Draggable from "vuedraggable";
+
+export default {
+  name: "PlaylistForm",
+  components: { Draggable },
+  props: { playlist: { type: Object, default: null } },
+  data() {
+    return {
+      newPlaylist: {
+        name: "",
+        description: "",
+        access: "shared",
+        playlist_type: null,
+        item_ids: [],
+      },
+      playlistTypes: [
+        {
+          value: "album",
+          text: this.$t("music.playlist.playlist_types.album"),
+        },
+        {
+          value: "artist",
+          text: this.$t("music.playlist.playlist_types.artist"),
+        },
+        {
+          value: "track",
+          text: this.$t("music.playlist.playlist_types.track"),
+        },
```

Comment:
I would put these in order of "small" to "large" (so tracks -> albums -> artists).